nixos/modules/services/networking/xrdp.nix

```diff
@@ -4,6 +4,37 @@ with lib;
 
 let
   cfg = config.services.xrdp;
+
+  upstreamProvidedConfigFilesToBeExposed = [
+    "km-00000406.ini"
+    "km-00000407.ini"
+    "km-00000409.ini"
+    "km-0000040a.ini"
+    "km-0000040b.ini"
+    "km-0000040c.ini"
+    "km-00000410.ini"
+    "km-00000411.ini"
+    "km-00000412.ini"
+    "km-00000414.ini"
+    "km-00000415.ini"
+    "km-00000416.ini"
+    "km-00000419.ini"
+    "km-0000041d.ini"
+    "km-00000807.ini"
+    "km-00000809.ini"
+    "km-0000080a.ini"
+    "km-0000080c.ini"
+    "km-00000813.ini"
+    "km-00000816.ini"
+    "km-0000100c.ini"
+    "km-00010409.ini"
+    "km-19360409.ini"
+    "sesman.ini"
+    "startwm.sh"
+    "xrdp.ini"
+    "xrdp_keyboard.ini"
```

Comment:
From where did you get that list?

Reply:
Looking at the code, it seems there are 2 possible other ways to fix this:
- The code for `environment.etc` supports glob that seems to be not documented anywhere: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/etc/etc.nix#L24-L30
- We also have a `environment.etc.<name>.mode` (https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/etc/etc.nix#L122-L131) that when set will copy the files instead of symlink

I think using one of those 2 options or even both should allow us to add those files to the correct place without hardcoding the files.